cypress/integration/upload-download.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { UploadPage } from "../page/index.ts";
```

Comment:
As a tip, you can simply do `... from "../page"` and it'll work the same way. If you want even more ergonomic paths, you can also see: https://www.typescriptlang.org/tsconfig#paths